logic/auth.js

```diff
@@ -218,9 +255,10 @@ async function register(user, seed) {
         throw new NodeError('Unable to encrypt mnemonic seed');
     }
 
-    //save user
+    //save user and init settings
     try {
         await diskLogic.writeUserFile({ name: user.name, password: user.password, seed: encryptedSeed });
+        await diskLogic.writeSettingsFile({});
```

Comment:
Is there a reason we create a new `settings.json` file here instead of using `user.json`?

Settings should probably be a nested object inside the user object like:

```
{
  "name": "lukechilds",
  "password": "password-hash",
  "seed": "encrypted-seed",
  "settings": {
    "currency": "USD",
    "foo": "bar"
  }
}
```

Reply:
There's not particular reasons, I just found two functions `writeSettingsFile` and `readSettingsFile` already written in diskLogic, so I thought you'd added then to use a settings file later.

Reply:
Ohhh sorry I missed that, I understand now. I think maybe that's a relic from the old Casa code.